hs_discover/public/index.html

```diff
@@ -127,11 +126,6 @@ <h4 class="panel-title">
                                             <small>Public</small>
                                         </div>
 
-                                        <div class="col-sm-6">
-                                            <img src="{{ STATIC_URL }}img/private.png" alt="Private Resource"/>
-                                            <small>Private</small>
-                                        </div>
-
                                         <div class="col-sm-6">
```

Comment:
Private resources aren't displayed in Discover so shouldn't be part of the legend